tests/docker/build.sh

```diff
@@ -54,16 +54,16 @@ ninja install
 cd
 git clone --single-branch https://chromium.googlesource.com/codecs/libgav1
 cd libgav1
-git checkout 45a1d76
-git clone https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp
+git checkout b712ad2
+git clone  --single-branch https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp
```

Comment:
We could also use the latest LTS tag 20200923.2.

Reply:
Great idea! Instead of the tag I switched over to the lts_2020_09_23 maintenance branch to include the latest (and future) bug fixes.